src/Uno.UI/Extensions/ViewExtensions.wasm.cs

```diff
@@ -18,8 +17,8 @@ public static partial class ViewExtensions
 		/// <param name="childLevelLimit">Defines the max depth, null if not limit (Should never be used)</param>
 		/// <param name="includeCurrent">Indicates if the current view should also be tested or not.</param>
 		/// <returns></returns>
-		public static T FindFirstChild<T>(this ViewGroup view, int? childLevelLimit = null, bool includeCurrent = true)
-			where T : View
+		public static T FindFirstChild<T>(this UIElement view, int? childLevelLimit = null, bool includeCurrent = true)
```

Comment:
breaking change ?

Reply:
Is it? Is the use of a type alias exposed publicly?